gulpfile.js/copy-asp-files.js

```diff
@@ -33,29 +34,57 @@ function restorePackages(callback) {
   callback();
 }
 
-exports.copyAspFiles = series(
+task('prepareMisc', parallel(
+  () => src('MVCDemos/AppData/*.ldf', { read: false }).pipe(clean()),
+  () => src('NetCoreDemos/Northwind.mdf')
+    .pipe(dest('MVCDemos/App_Data')),
+
+  () => src('NetCoreDemos/project.lock.json', { read: false, allowEmpty: true }).pipe(clean()),
+
+  () => src('SampleImages/**/*')
+    .pipe(dest('MVCDemos/Content/SampleData/SampleImages'))
+    .pipe(dest('NetCoreDemos/wwwroot/SampleData/SampleImages')),
+));
+
+exports.buildAndCopyAspFiles = series(
   checkAspnetPath,
   runBuild,
   parallel(
-    () => src('MVCDemos/AppData/*.ldf', { read: false }).pipe(clean()),
-    () => src('NetCoreDemos/Northwind.mdf')
-      .pipe(dest('MVCDemos/App_Data')),
+    'prepareMisc',
 
-    () => src('NetCoreDemos/project.lock.json', { read: false, allowEmpty: true }).pipe(clean()),
+    () => src('artifacts/*.{dll,pdb,xml}', { cwd: aspnetPath })
+      .pipe(dest('MVCDemos/lib')),
+
+    () => src('artifacts/*.nupkg', { cwd: aspnetPath })
+      .pipe(dest('NetCoreDemos/Nuget')),
```

Comment:
**Minor Suggestion**

It is probably worth extracting a task-building function, accepting path as an argument ( aspnetPath + 'artifacts' in one case and aspnetPackagesPath in another)